alicloud/resource_alicloud_oss_bucket_test.go

```diff
@@ -235,6 +229,29 @@ func testAccCheckOssBucketExistsWithProviders(n string, b *oss.BucketInfo, provi
 	}
 }
 
+func testAccCheckOssBucketLocationIsLocal(bucketInfo *oss.BucketInfo) resource.TestCheckFunc {
+	providers := []*schema.Provider{testAccProvider}
+
+	return func(s *terraform.State) error {
+		for _, provider := range providers {
+			// Ignore if Meta is empty, this can happen for validation providers
+			if provider.Meta() == nil {
+				continue
+			}
+
+			client := provider.Meta().(*AliyunClient)
+			localLocation := "oss-" + client.RegionId
+			if bucketInfo.Location == localLocation {
+				return nil
+			}
+
+			return fmt.Errorf("bucket location is %v but must be %v", bucketInfo.Location, localLocation)
+		}
+
+		return fmt.Errorf("unable to find the local bucket location")
+	}
+}
+
```

Comment:
What does this method use to do?

Reply:
The test TestAccAlicloudOssBucketBasic failed when I executed it in Frankfurt (eu-central-1), because it checked that the bucket location was `oss-cn-beijing`, but the correct value for me was `oss-eu-central-1`.
The function testAccCheckOssBucketLocationIsLocal checks that the bucket location is `oss-provider_region`.

Reply:
Hi @marcplouhinec I think this method is needless and I have improved this test case. Can you remove this commit? After that I will merge this PR.